src/anyvar/anyvar.py

```diff
@@ -19,6 +20,10 @@
 from anyvar.translate.vrs_python import VrsPythonTranslator
 from anyvar.utils.types import VrsObject
 
+# Suppress pydantic warnings unless otherwise indicated
+if os.environ.get("ANYVAR_SHOW_PYDANTIC_WARNINGS", None) is None:
```

Comment:
I kind of thought there was a linting rule about not supplying `None` as an arg to `get()` since it's the default but I guess not?